src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js

```diff
@@ -8,6 +10,46 @@
     const btns = [...doc.querySelectorAll('.ibexa-btn--extra-actions')];
     const menu = doc.querySelector('.ibexa-context-menu');
     const backdrop = new ibexa.core.Backdrop();
+    const formsInitialData = new Map();
+    const saveInitialFormData = (extraActionsContainer) => {
+        const extraActionsInputs = extraActionsContainer.querySelectorAll('input, select');
+
+        extraActionsInputs.forEach((node) => {
+            const value = node.type === 'radio' || node.type === 'checkbox' ? node.checked : node.value;
+
+            formsInitialData.set(node, value);
+        });
+    };
+    const restoreInitialFormData = (extraActionsContainer) => {
```

Comment:
Will `form.reset()` not work? https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset

Reply:
Unfortunately not, I was looking into it, but for some strange reason, `form.reset()` doesn't trigger any events on inputs (mainly `change` ones) :/